MYabrv.bib

```diff
@@ -48,6 +48,7 @@ @String{FormaliSE
 @String{FORTE = "Proc.\ Int'l Conf.\ on Formal Techniques for Distributed Objects, Components, and Systems (FORTE)"}
 @String{FoVeOOS = "Proc.\ Int'l Conf.\ on Formal Verification of Object-Oriented Software (FoVeOOS)"}
 @String{FSE24 = "Proc.\ Int'l Conference on the Foundations of Software Engineering (FSE)"}
+@String{FSE24Companion = "Companion Proc.\ Int'l Conference on the Foundations of Software Engineering (FSE Companion)}"}
```

Comment:
Too many closing brackets here. This is probably also causing the error in the CI tests

Reply:
thanks, good catch, fixed